tools/ttmlir-opt/CMakeLists.txt

```diff
@@ -26,13 +38,36 @@ VhloTypes)
 
 get_property(dialect_libs GLOBAL PROPERTY MLIR_DIALECT_LIBS)
 get_property(conversion_libs GLOBAL PROPERTY MLIR_CONVERSION_LIBS)
-set(LIBS ${stablehlo_libs} ${dialect_libs} ${conversion_libs} MLIROptLib MLIRTargetCpp TTMLIRStatic)
+set(LIBS TTMLIRStableHLOToTTIR TTMLIRCAPI ${stablehlo_libs} ${dialect_libs} ${conversion_libs} MLIROptLib MLIRTargetCpp 
+MLIRTTIRAnalysis
+TTMLIRTTIRToTTNN
+MLIRTTNNTransforms
+TTMLIRTosaToTTIR
+TTMLIRTTNNToEmitC
+MLIRTTMetalTransforms
+MLIRTTIRTransforms
+MLIRTTMetalDialect
+MLIRTTKernelDialect
+MLIRTTIRDialect
+MLIRTTNNDialect
+MLIRTTNNPipelines
+MLIRTTDialect
+TTNNTargetFlatbuffer
+TTMLIRCAPI
+TTMLIRStableHLOToTTIR
+TTMLIRStatic)
```

Comment:
Why did we replace `TTMLIRStatic`?  `TTMLIRStatic` should contain all of this already